packages/icon/src/react/index.js

```diff
@@ -23,23 +23,22 @@ const style = {
 const IconNotFound = () => null
 
 class Icon extends React.PureComponent {
-  getIconById(id) {
-    const Comp = icons[id] ? icons[id](React, filterReactProps) : IconNotFound
-    return <Comp />
-  }
-
   render() {
-    const { id, ...props } = this.props
+    const { id, 'aria-label': ariaLabel, ...props } = this.props
```

Comment:
rename is needed, right?  (only to get a valid var name)

Reply:
yeah because `aria-label` isn't a valid var name